control_panel_api/tests/test_aws_s3_access_policy.py

```diff
@@ -116,10 +120,8 @@ def test_revoking_access(self):
 
         document = policy.document
         self.assert_can_access_console(document)
-        for arn in [readwrite_arn, readwrite_arn]:
```

Comment:
uh, I didn't notice I made this mistake (`readwrite_arn, readwrite_arn`). Anyway, it's gone now as I got rid of the loop.